stactools_pipelines/pipelines/icesat2_boreal/requirements.txt

```diff
@@ -0,0 +1,5 @@
+git+https://github.com/MAAP-project/icesat2-boreal-stac.git@feat/v2.1
```

Comment:
Want to update this to main after merging the other branch or do you keep the branch to make the changes to the previous version?

Reply:
I'll tag the other repo after we merge that PR then update this to use the tag ref!